README.md

```diff
@@ -54,20 +54,37 @@ cd YimMenu
 
 #### Generate project files:
 
-```bash
-mkdir build && cd build
-cmake ..
-```
-Now, you will be able to open the solution, and simply build it in Visual Studio.
+- On Windows
+
+  - Visual Studio
+
+    If you only use Visual Studio and don't want to mess with command lines, Visual Studio have a CMake extension that does all the work.
+
+    Make sure it is [installed](https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170#installation).
+
+    Then, just open Visual Studio, Open a local folder, and select the `YimMenu` folder that the `git clone` command just created.
+
+    Let the CMake extension generate the needed files for building, you can follow the progress in the Output tab of Visual Studio.
+
+    Then, you can just build by using the Build menu at the top and clicking Build All.
+
+- Other OSs / IDEs
+
+  If you use something else, just follow this:
+
+  ```bash
+  mkdir build && cd build
+  cmake ..
+  ```
+
+  Now, you will be able to open the solution, and build it.
 
 
 ## Staying Up To Date
 
-```bash
-git pull https://github.com/YimMenu/YimMenu.git
```

Comment:
But CMake doesn't pull YimMenu changes for you.

Reply:
`git pull <url>` isn't a valid command either.
Just use `git pull`

Though it should still be included for those fetching from remote.

Reply:
This is an instruction to stay up to date which people would probably use when they make a fork, so having remote in `git pull` should help them to merge.

Reply:
The keeping up-to-date section was for people who were just git cloning YimMenu locally and building (was also written before we built DLL's automatically and published them as a release).
It was never written in a way for forks to stay up-to-date.